pkgs/top-level/all-packages.nix

```diff
@@ -12305,7 +12305,8 @@ with pkgs;
 
   softether_4_18 = callPackage ../servers/softether/4.18.nix { };
   softether_4_20 = callPackage ../servers/softether/4.20.nix { };
-  softether = softether_4_20;
+  softether_4_25 = callPackage ../servers/softether/4.25.nix { };
+  softether = softether_4_25;
```

Comment:
Why do we have all these old versions?

Reply:
17.09 branch does not contain version 4.20 that does not sure the config files that works will